src/utilities/animation-utility.js

```diff
@@ -0,0 +1,41 @@
+export const getTargetLeft = (touchOffset, slide, props, state) => {
```

Comment:
@ashika01 @sarmeyer Me and Ashika were talking about moving this function back to Carousel class since it is used so many times in the class component. After looking it over a bit and asking around I would like to keep it in this `animation-utility.js` file. This function is taken in some arguments and returning a calculation based off those arguments. It doesn't need to belong to the class itself. In theory, this function could calculate the same thing for any class component that would need it with out know about the class component itself.  Since the main purpose for this branch is to move as many function as possible to a separate file to make the `index.js` readable I would like to keep this in the `animation-utility.js` file. I would like to another option about this and would like to hear the best practice for something like this.